db/external_sst_file_ingestion_job.cc

```diff
@@ -739,6 +739,9 @@ Status ExternalSstFileIngestionJob::AssignLevelAndSeqnoForIngestedFile(
   if (force_global_seqno) {
     *assigned_seqno = last_seqno + 1;
     if (compaction_style == kCompactionStyleUniversal || files_overlap_) {
+      if (ingestion_options_.fail_if_not_bottommost_level) {
+        return Status::InvalidArgument("Files cannot be ingested to Lmax");
```

Comment:
Hm but the argument is valid. Maybe Incomplete or TryAgain? I actually like TryAgain assuming the error indicates the bottom-level range needs to be cleared.

Reply:
Maybe TryAgain would make more sense coupled with an API comment on `fail_if_not_bottommost_level` that the user is responsible for ensuring the bottom-level is clear in the range. Anyways, up to you.

Reply:
Done.